libcxx/docs/ReleaseNotes/18.rst

```diff
@@ -81,6 +81,13 @@ Improvements and New Features
 - The ``_LIBCPP_ENABLE_CXX26_REMOVED_STRING_RESERVE`` macro has been added to make
   the function ``std::basic_string<...>::reserve()`` available.
 
+- The experimental standard library modules can now be installed. By default,
+  they are not installed. This can be enabled by configuring CMake with
+  ``-DLIBCXX_INSTALL_MODULES=ON``. The installation directory can be configured
+  with the CMake option ``-DLIBCXX_INSTALL_MODULE_DIR=<path>``. The default
+  location is ``modules/c++/v1`` in the same prefix as the ``include``
+  directory.
```

Comment:
```suggestion
- The experimental standard library C++20 module interface unit source files can
  now be installed. By default, they are not installed. This can be enabled by
  configuring CMake with ``-DLIBCXX_INSTALL_MODULE_INTERFACE_UNIT_SOURCES=ON``.
  The installation directory can be configured with the CMake option
  ``-DLIBCXX_INSTALL_MODULE_INTERFACE_UNIT_SOURCE_DIR=<path>``. The default
  location is ``modules/c++/v1`` in the same prefix as the ``include`` directory.
```
The location in that last sentence doesn't appear to be consistent with the default specified in `libcxx/CMakeLists.txt`.

Reply:
I noticed that too, that was an older proposed location.